README.md

```diff
@@ -293,9 +335,11 @@ interface IAuthenticationService {
 
 	/**
 	 * Invalidates the current user authentication data.
+	 * If options.openAction is provided, it will be used to open the logout url instead of the default opener.
+	 * @param {ILoginOptions} options Optional settings for the logout method.
```

Comment:
The JSDocs says the param is of `ILoginOptions` type and the actual method's arg is marked as `IOpenActionOptions`
I know one inherits the other but it'd still be nice to be consistent